src/components/screens/consul/Proposals/NewProposal.js

```diff
@@ -239,6 +284,16 @@ export const NewProposal = ({ navigation, data, query }) => {
               </ScrollView>
             </>
           )}
+
+          {item.type === ITEM_TYPES.PICKER && (
+            <Controller
+              name={item.name}
+              control={control}
+              render={({ onChange, value }) => (
+                <ImageSelector {...{ control, imageUri, item, onChange, selectImage, value }} />
```

Comment:
If I use it like this, `value` will not get any empty value after the image is selected once. The user will not be able to change the image as it will be filled with `imageUri`. I need to think of something else for this.

I need `value`. I need to fetch as `image: "xxx"` in `onSubmit` function. I can only achieve this with `onChange`. Am I wrong?

Reply:
0bc389056155a1878daf189d621c7353bcc70657